.github/workflows/release.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v3.0.0
```

Comment:
Should we update this also to v6. Check example here - https://goreleaser.com/ci/actions/#workflow

Reply:
Based on the release notes with v6, by default it uses v2. 
https://goreleaser.com/blog/goreleaser-v2/#github-action